benchflow-test-manager/application/src/test/java/cloud/benchflow/testmanager/scheduler/TestTaskSchedulerIT.java

```diff
@@ -267,6 +270,60 @@ public void runLoadTest() throws Exception {
 
   }
 
+  @Test
+  public void runBenchFlowTestTimeoutTest() throws Exception {
```

Comment:
You need also some tests where you submit multiple tests, and check the results of some of them, so that you can experience what happens with concurrency